build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -559,6 +569,12 @@ message Command {
   // the client SHOULD ensure that running the action on any such worker will
   // have the same result.
   // A detailed lexicon for this can be found in the accompanying platform.md.
+  // DEPRECATED as of v2.2: platform properties are now specified directly in
+  // the action to allow execution services to make routing decisions based on
+  // these without introducing an additional step to reach the CAS to determine
+  // the properties. See documentation note in the
+  // [Action][build.bazel.remote.execution.v2.Action] for migration.
+  // This will be deprecated in v3.
```

Comment:
Presumably this will actually be removed in v3, not deprecated. Regardless, I don't think it's necessary to say that explicitly here.